src/main/java/pl/javastart/task/MixPhoneContract.java

```diff
@@ -0,0 +1,53 @@
+package pl.javastart.task;
+
+public class MixPhoneContract implements Phone {
```

Comment:
Mix ma wiele wspólnego z umową na kartę. Najpierw wykorzystujemy pakiet smsów, mmsów i minut, a jeżeli już je zużyjemy, to powinno nam pobierać środki z konta. Powinieneś więc tutaj dziedziczyć po czymś innym